src/Umbraco.Infrastructure/Security/IBackOfficeApplicationManager.cs

```diff
@@ -2,8 +2,12 @@
 
 public interface IBackOfficeApplicationManager
 {
+    [Obsolete("Please use the overload that allows for multiple back-office hosts. Will be removed in V17.")]
     Task EnsureBackOfficeApplicationAsync(Uri backOfficeUrl, CancellationToken cancellationToken = default);
 
+    Task EnsureBackOfficeApplicationAsync(IEnumerable<Uri> backOfficeHosts, CancellationToken cancellationToken = default)
+        => Task.CompletedTask;
```

Comment:
Should this default implementation really be iterating the collection and calling the obsolete method?

Reply:
Either way would be functionally wrong, I think. Iterating the collection would mean us ending up in a "last in wins" scenario. The default implementation can (will) be removed when the obsoleted method is removed so it's only a temporary measure anyway.

Reply:
Ah, true - I was thinking that would replicate the behaviour we now do want, but you're right, it won't.